src/Shouldly.Tests/ShouldBe/HalfScenarios.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NET5_0_OR_GREATER
+﻿#if NET8_0_OR_GREATER
```

Comment:
I like to keep these "or greater" checks as documentation for the exact .NET version in which the type appeared (5.0).